blocks/core/ff_container/ff_container-dashboard-section/ff_container-dashboard-section.less

```diff
@@ -0,0 +1,32 @@
+//header
+.ff_container-dashboard-section__header {
+    padding: @ff_size_control_vertical;
```

Comment:
I think we need a different var setup. In the new style guide, there's three situations - the 'Box' has 10px 20px, the tab box has 15px 20px 20px 20px, the filter box has all 15px. So probably a var for each value, and three shortcut vars that combine them in each situation?